backend/pom.xml

```diff
@@ -97,8 +97,16 @@
 			<artifactId>modelmapper</artifactId>
```

Comment:
Have these already been added in @nasaku898  pull request to add tests?

Reply:
If you're referring to JUnit, I'm not sure. I see some other different `junit` dependencies but I needed to use `junit` to be able to use Mockito's `@InjectMocks` to mock the `PictureService` in my test (in the other test file, I saw that the service was directly instantiated, i.e. `new WhateverService();`, but I don't think that's quite proper)

Reply:
Nvm, I see you are referring to the other PR. I will wait until it is merged and fix my current code.